code/__defines/input.dm

```diff
@@ -0,0 +1,21 @@
+// Bitflags for the move_keys_held bitfield.
+#define NORTH_KEY   (1<<0)
+#define SOUTH_KEY   (1<<1)
+#define EAST_KEY    (1<<2)
+#define WEST_KEY    (1<<3)
+#define W_KEY       (1<<4)
+#define S_KEY       (1<<5)
+#define D_KEY       (1<<6)
+#define A_KEY       (1<<7)
```

Comment:
Challenge: Why do we need to track WASD separately from NESW? It adds a lot of logical complexity that makes this stuff harder to read. If we drop it, the only change I see is that if you're holding both UP and W, the first one you release will stop movement, and I don't necessarily see that as a bad thing.

Reply:
SSinput allows for keybinding in the future and it is generally more robust for minimal additional complexity. I don't think it's significantly harder to read personally.